src/App.tsx

```diff
@@ -15,71 +16,108 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  name: string;
+  selectedItem: boolean;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    name: 'Jam',
+    selectedItem: true,
```

Comment:
You don't need 2 states here. Just use one that can be string or an empty for example
```suggestion
    selectedGood: 'Jam',
```